tensorrt_llm/_torch/autotuner.py

```diff
@@ -1096,8 +1117,7 @@ def _optimization_profiles(
             # Add the current input value as one of the opt values
             opt_shapes = set(opt_shapes)
             opt_shapes.add(
-                spec.map_to_tuning_buckets(
-                    base_profile.shapes[spec.input_idx][spec.dim_idx].val))
+                base_profile.shapes[spec.input_idx][spec.dim_idx].val)
```

Comment:
@hyukn As discussed offline, I have removed bucket mapping during tuning. The mapping is only applied at inference time now.